### PR TITLE
Replace build tests with build testing

### PR DESCRIPTION
**Pull request by oerc0122**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -44,15 +41,15 @@ include(PACE_Version)
 set(PACE_MLINT_IGNORE ${PACE_MLINT_IGNORE} "${Herbert_ROOT}/**/*.m")
 
 include(PACE_CodeAnalysis)
-if(${BUILD_TESTS})
-    include(CTest)
+include(CTest)
+if(BUILD_TESTING)
```
Comment (cmarooney-stfc) [inline CMakeLists.txt:45]: Presuming that BUILD_TESTING is not required to be a $variable but checking - I can see it comes from the -D options below but so did $BUILD_TESTS

Comment (oerc0122) [reply CMakeLists.txt:45]: If is an odd cookie and doesn't quire the expansion manually. Purely stylistic, but it's the way I usually see ifs. 
https://cmake.org/cmake/help/latest/command/if.html#variable-expansion

**Comment by abuts** [comment]
have no opinion on this -- may be will have after cmake course

**Review by abuts — COMMENTED** [review]
have no opinion on this -- may be will have after cmake course

**Comment by oerc0122** [comment]
Essentially the issue is that `BUILD_TESTS` sets `BUILD_TESTING` to on anyway (`enable_testing()` sets this flag as it's the standard CMake/CTest flag) this just means that we are using the CMake standard flag people will expect rather than one of our own designation.

**Pull request merged**